applications/serial_lte_modem/src/main.c

```diff
@@ -91,7 +94,11 @@ static void exit_idle(struct k_work *work)
 {
 	int err;
 
+	if (k_sem_take(&semWake, K_NO_WAIT) != 0)
+		return;
+
```

Comment:
This is workaround not solution for root cause.
It might be due to we remove the callback not from callback handler, but work handler.
Recommend to simply move the remove to the callback function.

Reply:
Even inside handler ,to remove callback is not fast enough.
I can easily reproduce double wakeup with remove callback inside handler method.

Reply:
What's the toggle interval and what's the return of remove_callback API?